ipywebrtc/webrtc.py

```diff
@@ -402,6 +402,7 @@ class VideoRecorder(Recorder):
     _view_name = Unicode('VideoRecorderView').tag(sync=True)
 
     video = Instance(Video).tag(sync=True, **widget_serialization)
+    codecs = Unicode('', help='Optional codecs for the recording, e.g. "vp8" or "vp9, opus".').tag(sync=True)
```

Comment:
my guess it should be with space?

Reply:
since this is passed with quotes around it, the space is allowed.